test_all.py

```diff
@@ -18,3 +23,9 @@ def test_mainwindow_and_frames_contains_a_name(self):
             self.instance.staff_meeting_frame['text'],
             "Zespół orzekający"
         )
+
+    def test_School_class_contains_only_sort_of_sorts_list(self):
+        query = School.select()
```

Comment:
The whole test here You can write in 2 lines:
```
for school in School.select():
    self.assertIn(school.sort, sorts)
```
it will do exactly the same thing.